### ISSUE TITLE
tronscan mac won't send trx

### ISSUE THREAD

**Issue by stev3nb** [tronscan/tronscan-frontend#826]
getting a 'something went wrong' message over and over when trying to send

**Comment by cccyyc** [comment]
Are you using desktop or explorer?

**Comment by stev3nb** [comment]
Desktop

On 28 Nov 2018, at 7:36 pm, cccyyc <notifications@github.com> wrote:

Are you using desktop or explorer?

**Comment by paulmeade7** [comment]
I've got the same problem. My Tronscan Desktop Mac won't let me send my TRX that's sitting on my Ledger Nano S over to my regular Wallet to be able to trade on DEX. I'm signed into both and it won't work. Just keeps hourglassing.

**Comment by stev3nb** [comment]
Yes exactly. Looks like the comms between the Ledger and Tronscan Desktop app is flakey. I contacted the developer on Twitter but he hasn’t replied. I eventually got all my TRX out by trying over and over until it worked. Now using the Tronscan Web app with DEX and seems to work fine now. It was a stressful experience, and I work in the IT industry. Wonder how non technical users are coping with all this nonsense.

Sent from mobile

On 29 Nov 2018, at 6:04 am, paulmeade7 <notifications@github.com> wrote:

I've got the same problem. My Tronscan Desktop Mac won't let me send my TRX that's sitting on my Ledger Nano S over to my regular Wallet to be able to trade on DEX. I'm signed into both and it won't work. Just keeps hourglassing.
